Frontend/ui-driver/src/Screens/HomeScreen/Controller.purs

```diff
@@ -586,7 +591,9 @@ eval (TimeUpdate time lat lng) state = do
     pure AfterRender
     ]
 
-eval (RideActiveAction activeRide) state = updateAndExit state { data {activeRide = activeRideDetail state activeRide}} $ UpdateStage ST.RideAccepted state { data {activeRide = activeRideDetail state activeRide}}
+eval (RideActiveAction activeRide) state = do
+  let currActiveRideDetails = activeRideDetail state activeRide
```

Comment:
create a local variable and update the state and use that variable. instead of doing samething twice.